src/basicview.cpp

```diff
@@ -152,14 +155,16 @@ void BasicView::update(Cluster cluster)
     bool isRunning = cluster.status() == "Running";
     bool isPaused = cluster.status() == "Paused";
     topStatus->setText(cluster.status());
+    serviceButton->setEnabled(isRunning || isPaused);
     stopButton->setEnabled(isRunning || isPaused);
     pauseButton->setEnabled(isRunning || isPaused);
     mountButton->setEnabled(isRunning || isPaused);
+    tunnelButton->setEnabled(isRunning || isPaused);
     deleteButton->setEnabled(exists);
     dashboardButton->setEnabled(isRunning);
 #if __linux__ || __APPLE__
-    dockerEnvButton->setEnabled(isRunning);
-    sshButton->setEnabled(exists);
+    dockerEnvButton->setEnabled(isRunning || isPaused);
+    sshButton->setEnabled(isRunning || isPaused);
 #else
```

Comment:
The above `tunnelButton` and `mountButton` enable lines above should go in here as we don't currently support opening the terminal in Windows, so we shouldn't allow Windows to click the buttons as they both open the terminal.

Reply:
done